puppet/modules/jenkins_job_builder/manifests/config.pp

```diff
@@ -41,7 +41,7 @@
   }
 
   exec { "remove_unmanaged_jobs-${config_name}":
-    command => "jenkins-jobs --conf ${inifile} delete --jobs-only $(ruby ${directory}/${config_name}/unmanaged_jobs.rb ${inifile})",
+    command => "${directory}/${config_name}/unmanaged_jobs.rb ${inifile}",
```

Comment:
shouldn't it be `ruby ${directory}/...` ?

Reply:
You might be right, not sure if the file is marked as executable.

Reply:
Updated